src/components/LessonListsByStates/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import {
+  newLessonsActionText,
+  inProgressTitleText,
+  inQueueTitleText,
+  publishedTitleText,
+  noInProgressLessonsDescriptionText,
+  noInQueueLessonsDescriptionText,
+  noPublishedLessonsDescriptionText,
+} from 'utils/text'
+import {inProgressLessonStates, publishedLessonStates} from 'utils/lessonStates'
+import LessonList from 'components/LessonList'
+import Tabs from 'components/Tabs'
+import Prompt from 'components/Prompt'
+
+export default ({instructor}) => (
+  <Tabs groups={[
+    {
+      title: inProgressTitleText,
+      component: (
+        <LessonList
+          states={inProgressLessonStates}
+          fallback={
+            <div className='pa3'>
+              <Prompt
+                description={noInProgressLessonsDescriptionText}
+                actionText={newLessonsActionText}
+                action={'/lessons/new'}
+              />
+            </div>
+          }
+          instructor={instructor}
+        />
+      ),
+    },
+    {
+      title: inQueueTitleText,
+      component: (
+        <LessonList
+          states={['approved']}
+          fallback={
+            <div className='pa3'>
+              <Prompt
+                description={noInQueueLessonsDescriptionText}
+                actionText={newLessonsActionText}
+                action={'/lessons/new'}
+              />
+            </div>
+          }
+          instructor={instructor}
+        />
+      ),
+    },
+    {
+      title: publishedTitleText,
+      component: (
+        <LessonList
+          states={publishedLessonStates}
+          fallback={
+            <div className='pa3'>
+              <Prompt
+                description={noPublishedLessonsDescriptionText}
+                actionText={newLessonsActionText}
+                action={'/lessons/new'}
+              />
+            </div>
+          }
+          instructor={instructor}
+        />
+      ),
+    },
+  ]} />
+)
```

Comment:
This creates the tabbed LessonLists for `Instructor` (Overview view) and `Lessons` (Lessons view):
![image](https://cloud.githubusercontent.com/assets/5497885/23436647/835013d8-fdc8-11e6-957a-37e2af49eb22.png)
![image](https://cloud.githubusercontent.com/assets/5497885/23436657/8d1df06a-fdc8-11e6-9472-3a802a87e772.png)